docs/plans/2026-08-02-persistent-generated-data-spec.md

```
@@ -3,7 +3,7 @@ search:
   exclude: true
 ---
 
-Status: Approved specification; execution authority is the [active implementation plan](2026-08-02-persistent-generated-data-implementation-plan.md)
+Status: Implemented historical specification; current behavior is documented in the present-state architecture and CLI contract
```

markdownlint-cli2 (0.23.2)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)